app/boards/shields/nibble/nibble.keymap

```diff
@@ -8,27 +8,40 @@
 #include <dt-bindings/zmk/keys.h>
 #include <dt-bindings/zmk/bt.h>
 
-#define DEFAULT 0
-#define FUNC 1
+#define LAYER_ID_DEFAULT 0
+#define LAYER_ID_FUNC 1
```

Comment:
I've never seen `LAYER_ID` before. Is this something we are doing now? I assume it's to avoid conflicts with the layer name.

Reply:
This is a hold over from the original dev's approach. I'm not seeing it anywhere else and can remove it if preferred.

Reply:
I suppose it's up to you. It seems a bit odd to me, but there's nothing wrong about it.

Reply:
Agreed, do you prefer I go with the ZMK standard or the ported version that's pre-existing?

The only gain to keeping the pre-existing is it won't break keymaps based on the default that's present in main right now.

I'm good with either option.

Reply:
I believe the ones that are in main don't have LAYER_ID seeing as GitHub is marking this as a change, no?